docs/design/ovs-pipeline.md

```diff
@@ -406,42 +406,57 @@ In the future this table may support an additional mode of operations, in which
 installed in a table corresponding to that tier. The egress table to tier mappings 
 is as follows:
 
 ```text
-Application tier -> DefaultTierEgressRuleTable(49)
-All other tiers  -> MultiTierEgressRuleTable(45)
+Baseline Tier   ->  EgressDefaultTable(60)
+All other Tiers ->  AntreaPolicyEgressRuleTable(45)
 ```
 
 Since the example ACNP resides in the Application tier, if you dump the flows for
-table 49, you should see something like this:
+table 45, you should see something like this:
 
 ```text
-1. table=49, priority=64990,ct_state=-new+est,ip actions=resubmit(,70)
-2. table=49, priority=58600,conj_id=2,ip actions=load:0x2->NXM_NX_REG6[],resubmit(,70)
-3. table=49, priority=58600,ip,nw_src=10.10.1.6 actions=conjunction(2,1/3)
-4. table=49, priority=58600,ip,nw_dst=10.10.1.8 actions=conjunction(2,2/3)
-5. table=49, priority=58600,udp,tp_dst=53 actions=conjunction(2,3/3)
-6. table=49, priority=0 actions=resubmit(,50)
+1. table=45, priority=64990,ct_state=-new+est,ip actions=resubmit(,61)
+2. table=45, priority=14000,conj_id=1,ip actions=load:0x1->NXM_NX_REG5[],ct(commit,table=61,zone=65520,exec(load:0x1->NXM_NX_CT_LABEL[32..63]))
+3. table=45, priority=14000,ip,nw_src=10.10.1.6 actions=conjunction(1,1/3)
+4. table=45, priority=14000,ip,nw_dst=10.10.1.8 actions=conjunction(1,2/3)
+5. table=45, priority=14000,udp,tp_dst=53 actions=conjunction(1,3/3)
+6. table=45, priority=0 actions=resubmit(,50)
 ```
+Notice how we use the OVS built-in `conjunction` action to implement policies
+efficiently. This enables us to do a conjunctive match across multiple
+dimensions (source IP, destination IP, port) efficiently without "exploding" the
+number of flows. By definition of a conjunctive match, we have at least 2
+dimensions. For our use-case we have at most 3 dimensions.
+
+The only requirements on `conj_id` is for it to be a unique 32-bit integer
+within the table. At the moment we use a single custom allocator, which is
+common to all tables that can have NetworkPolicy flows installed (45, 50,
+60, 85, 90 and 100). This is why `conj_id` is set to 1 in the above example
+(1 was allocated for the egress rule of our K8s NetworkPolicy example).
 
-Similar to K8s NetworkPolicy implementation, AntreaPolicyEgressRuleTables also rely on
-the OVS built-in `conjunction` action to implement policies efficiently.
 The above example flows read as follow: if the source IP address is in set
 {10.10.1.6}, and the destination IP address is in the set {10.10.1.8}, and the
 destination TCP port is in the set {53}, then use the `conjunction` action with
-id 2, which goes to [L3ForwardingTable]. Otherwise, go to the table corresponding
-to the next tier in precedence, or in the case of the Application tier (lowest
-precedence), go to the [EgressRuleTable].
+id 1, which stores the `conj_id` 1 in `ct_label[32..63]` for egress metrics collection
+purposes, and forwards the packet to EgressMetricsTable, then [L3ForwardingTable].
```

Comment:
the doc hasn't been updated to account for EgressMetricsTable & IngressMetricsTable yet?

Reply:
Yes. I was hoping @tnqn can follow up on that regard, also check what I've added here is accurate.